### PR TITLE
Adopt to the 4.3 Add custom assembly 

### PR DESCRIPTION
**Pull request by vparfonov**

### CODE REVIEW

**Comment by vparfonov** [comment]
@ashumilova @sleshchenko

File: assembly/assembly-ide-war/.codenvy/classpath [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sleshchenko) [inline assembly/assembly-ide-war/.codenvy/classpath]: Let's remove this file

File: assembly/assembly-ide-war/pom.xml [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-openshift-assembly-parent</artifactId>
+        <groupId>org.eclipse.che</groupId>
+        <version>4.3.0-RC1-SNAPSHOT</version>
+    </parent>
+    <artifactId>openshift-plugin-assembly-ide-war</artifactId>
```
Comment (sleshchenko) [inline assembly/assembly-ide-war/pom.xml]: Maybe it would be better to name folder the same as artifact id 
@skabashnyuk WDYT?

Comment (vparfonov) [inline assembly/assembly-ide-war/pom.xml]: basicly +1, but in this case we need to use different groupId

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+# otherwise just creates the new workspace.
+workspace.runtime.auto_restore=false
+
+# OpenShift configuration
+openshift.api.endpoint=https://api.codenvy.openshift.com
+oauth.openshift.authuri=https://api.codenvy.openshift.com/oauth/authorize
+oauth.openshift.tokenuri=https://api.codenvy.openshift.com/oauth/token
+oauth.openshift.clientid=***
+oauth.openshift.clientsecret=***
+oauth.openshift.redirecturis=http://localhost:${SERVER_PORT}/che/api/oauth/callback
```
Comment (sleshchenko) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties]: Check that it is correct value

File: assembly/assembly-ide-war/pom.xml (hunk 2 above)
Comment (sleshchenko) [inline assembly/assembly-ide-war/pom.xml]: It is clear but why we can't rename folder?

**Comment by sleshchenko** [comment]
other LGTM

File: assembly/assembly-ide-war/pom.xml (hunk 2 above)
Comment (skabashnyuk) [inline assembly/assembly-ide-war/pom.xml]: +1 to rename folder. It's usual maven practive to have folder name and artifact id the same.
@TylerJewell any objections?

Comment (skabashnyuk) [inline assembly/assembly-ide-war/pom.xml]: BTW may be group ID should be different
For instance. org.eclipse.che.openshift or else.

File: assembly/openshift-plugin-assembly-main/.codenvy/classpath [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sleshchenko) [inline assembly/openshift-plugin-assembly-main/.codenvy/classpath]: Why do you need this file?

File: assembly/openshift-plugin-assembly-main/.codenvy/classpath [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+<classpath>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="src" path="src/test/java"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.che.MAVEN2_CLASSPATH_CONTAINER"/>
+</classpath>
```
Comment (skabashnyuk) [inline assembly/openshift-plugin-assembly-main/.codenvy/classpath]: new line

File: assembly/openshift-plugin-assembly-wsmaster-war/src/main/webapp/META-INF/context.xml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<Context allowCasualMultipartParsing="true">
+    <Valve className="org.apache.catalina.valves.rewrite.RewriteValve"/>
+</Context>
```
Comment (skabashnyuk) [inline assembly/openshift-plugin-assembly-wsmaster-war/src/main/webapp/META-INF/context.xml]: new line

**Comment by skabashnyuk** [comment]
other ok

**Pull request merged**